apps/article/stylesheets/amp.styl

```diff
@@ -0,0 +1,55 @@
+- amp-header-height = 54px
+
+.body-amp
+  max-width 580px
+
+  .amp-header
+    height amp-header-height
+    border-bottom 1px solid gray-lighter-color
+    display flex
+    align-items center
+    a
+      margin-left 20px
+
+  .article-header
+    margin-top 50px
+
+  amp-carousel
```

Comment:
Ah yeah that's a good point. They're currently scoped under .body-amp but that's pretty general still.